xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxFFmpeg.cpp

```diff
@@ -1715,6 +1716,21 @@ CDemuxStream* CDVDDemuxFFmpeg::AddStream(int streamIdx)
         if (!stereoMode.empty())
           st->stereo_mode = stereoMode;
 
+        if (m_bMatroska)
+        {
+          size_t size = 0;
+          uint8_t* side_data = nullptr;
```

Comment:
Please declare variables when they are used, no need to pre declare a variable here (I don't think we have a code guideline for this).

Reply:
Done.
Removed those. Compiled again. 
No errors here.